src/pages/TeamSettings/Members.vue

```diff
@@ -283,7 +291,12 @@ export default {
         icon="lock"
         max-width="600"
       >
-        <p>
+        <p v-if="isSelfServe">
+          Your team has no users available;
+          <router-link :to="'/plans'">upgrade your plan</router-link> to get
+          more users and access to more features!
```

Comment:
I think we want to be a little more explicit with this - the logic is correct but free/starter/standard are all self serve plans; I think changing the variable to `canUpgradeUsers` would be enough for maintainable clarity